packages/core/src/hooks/hookRunner.test.ts

```diff
@@ -263,16 +263,56 @@ describe('HookRunner', () => {
         );
 
         expect(spawn).toHaveBeenCalledWith(
-          '/test/project/hooks/test.sh',
+          expect.stringMatching(/bash|powershell|cmd/),
```

Comment:
The test expectation `expect.stringMatching(/bash|powershell|cmd/)` implies that cmd is a valid execution path. However, if the code forces PowerShell on Windows in shell-utils, cmd will never be matched, making this test assertion slightly misleading regarding the actual behavior. If cmd support is added, the test is fine. If cmd is intentionally dropped, remove cmd from the regex to reflect the reality.